### PR TITLE
fix pageworklaod

### LINKED ISSUE #5164: TiFlash pageworkload can't run the specified workload.
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
Compile TiFlash with `RELWITHDEBINFO ` in master branch, `./tiflash pageworkload --help`

### 2. What did you expect to see? (Required)
Show the supported workload like this:
image

### 3. What did you see instead (Required)
image

### 4. What is your TiFlash version? (Required)
617fe546febdf939c1a40aeefacbd22592ef757c

### PR DESCRIPTION
**Pull request by hehechen**
Signed-off-by: hehechen <awd123456sss@gmail.com>

### What problem does this PR solve?

Issue Number: close #5164

Problem Summary:

### What is changed and how it works?
Register workload in `mainEntry` explicitly.

image
### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
./tiflash pageworkload --help
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Storages/Page/workload/MainEntry.cpp [hunk 1]
```diff
@@ -24,31 +24,32 @@ int StressWorkload::mainEntry(int argc, char ** argv)
...
+        f();
         void _work_load_register_named_HeavyRead();
         f = _work_load_register_named_HeavyRead;
-        (void)f;
+        f();
         void _work_load_register_named_HeavySkewWriteRead();
         f = _work_load_register_named_HeavySkewWriteRead;
-        (void)f;
+        f();
+        ;
```
Comment (Lloyd-Pottiger) [inline dbms/src/Storages/Page/workload/MainEntry.cpp:34]: duplicate `;`

Comment (Lloyd-Pottiger) [reply dbms/src/Storages/Page/workload/MainEntry.cpp:34]: In this way, may be we can just remove `__attribute__((constructor))` in `#define REGISTER_WORKLOAD(WORKLOAD)` ?

**Review by Lloyd-Pottiger — APPROVED** [review]
LGTM

File: dbms/src/Storages/Page/workload/MainEntry.cpp [hunk 2]
```diff
@@ -24,31 +24,31 @@ int StressWorkload::mainEntry(int argc, char ** argv)
         // we do this to trigger REGISTER_WORKLOAD explicitly.
         void _work_load_register_named_HeavyMemoryCostInGC();
         void (*f)() = _work_load_register_named_HeavyMemoryCostInGC;
```
Comment (jiaqizho) [inline dbms/src/Storages/Page/workload/MainEntry.cpp:26]: can just `_work_load_register_named_HeavyMemoryCostInGC` replace it?

**Review by JaySon-Huang — APPROVED** [review]
lgtm

**Pull request merged**